recipes/polars/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "polars" %}
+{% set version = "0.12.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/polars-{{ version }}.tar.gz
+  sha256: 86978c91427b5f65aa21f070b4459b05bd55c2e5bcfbe8566f6398f48d5461c7
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  # script: {{ PYTHON }} -m pip install . -vv
+  script: maturin develop --rustc-extra-args="-C target-cpu=native" --release
```

Comment:
You shouldn't be building for the native CPU. This will lead to unportable binary.

Reply:
got it, that was the build script it supplied, I'll try something else